src/net.cpp

```diff
@@ -1215,14 +1217,8 @@ void CConnman::DisconnectNodes() {
             }
         }
     }
-    {
-        LOCK(cs_vNodes);
-        vNodesSize = vNodes.size();
-    }
-    if(vNodesSize != nPrevNodeCount) {
-        nPrevNodeCount = vNodesSize;
-        if(clientInterface)
-            clientInterface->NotifyNumConnectionsChanged(vNodesSize);
+    if(clientInterface && vNodesSize != nPrevNodeCount) {
+        clientInterface->NotifyNumConnectionsChanged(vNodesSize);
```

Comment:
Commit "Improve lock handling in DisconnectNodes."

Is it worth preventing notifying the same connection count?

Reply:
Yes, it triggers a notification that updates the GUI and would happen every 50ms.